src/test/modelinstance_test.cpp

```diff
@@ -381,6 +382,55 @@ TEST_F(TestLoadModel, CheckIfNonExistingXmlFileReturnsFileInvalid) {
     auto status = modelInstance.loadModel(config);
     EXPECT_EQ(status, ovms::StatusCode::FILE_INVALID) << status.string();
 }
+class TestLoadModelWithRemoteTensorFactoriesSucceeds : public ::testing::Test {
+protected:
+    ovms::ModelConfig config;
+    std::unique_ptr<ov::Core> ieCore;
+    void SetUp() {
+        ieCore = std::make_unique<ov::Core>();
+
+        config = ovms::ModelConfig{
+            "NOT_USED_NAME",
+            dummy_model_location,  // base path
+            "SOME",                // target device
+            "1",                   // batchsize
+            1,                     // NIREQ
+            false,                 // is stateful
+            false,                 // idle sequence cleanup enabled
+            false,                 // low latency transformation enabled
+            500,                   // stateful sequence max number,
+            "",                    // cache dir
+            1,                     // version
+            dummy_model_location,  // local path
+        };
+    }
+};
+TEST_F(TestLoadModelWithRemoteTensorFactoriesSucceeds, CheckIfLoadingSucceedsForGPU) {
+    SKIP_AND_EXIT_IF_NO_GPU();
```

Comment:
We agreed to contain OpenCL in the name if test is for gpu. And VAContext if it is VA related. @ngrozae